client/src/components/aboutUs/AboutUsBlock.js

```diff
@@ -24,11 +24,11 @@ var AboutUsBlock = React.forwardRef((props, ref) => {
                 <p>
                     For over 35 years, the University of Alberta's Computer Engineering Club has bought Computer Engineering students together, organized specialized events and initiatives, and advocated to larger bodies — such as the Engineering Students’ Society, the Faculty of Engineering, and the Students’ Union. In recent years however, we have expanded our scope to include students in Computer Science, and Data Science fields who share similar interests in industry.
                     We have a longstanding history of running large-scale events focused around educational and hands-on learning for programming and design, such as <a href="https://hacked.compeclub.com" target="_blank" rel="noopener noreferrer">HackED</a> and <a href="https://hackedbeta.compeclub.com" target="_blank" rel="noopener noreferrer">HackED Beta</a>. 
-                    Now, we are excited to continue this fairly new endeavor, with the <strong className="colored-text">goal of connecting undergraduate students to employers and businesses</strong> in a way that is unprecedented on our campus: Career++
+                    Now, we are excited to continue this fairly new endeavor, with the <strong className="colored-text">goal of connecting undergraduate students to employers and businesses</strong> in a way that is unprecedented on our campus: Aether
                 </p>
                 <p>
-                    While Career++ is still somewhat new, our club has phenomenal successful and experience running large-scale events. HackED is one of <strong className="colored-text">Alberta’s largest student-run hackathons with over 700 participants</strong>, and a history stretching back over 10 years. 
-                    Additionally, at the inaugeral Career++ last year, we had an incredible participation rate of <strong className="colored-text">over 1000 students</strong> and received <strong className="colored-text">overwhelmingly positive feedback</strong> from the students and companies! 
+                    While Aether is still somewhat new, our club has phenomenal successful and experience running large-scale events. One of which is HackED, <strong className="colored-text">Alberta’s largest student-run hackathons with over 700 participants</strong>, and a history stretching back over 10 years. 
+                    Additionally, at the inaugeral Aether(previously Career++) last year, we had an incredible participation rate of <strong className="colored-text">over 2000 students</strong> and received <strong className="colored-text">overwhelmingly positive feedback</strong> from the students and companies! 
```

Comment:
I think we only had 1200 participants last year, and a potential reach/community of 2300 EE, CS, and CE students